.azuredevops/modulePipelines/ms.app.managedenvironments.yml

```diff
@@ -0,0 +1,147 @@
+name: 'Container Apps: Managed Environments'
+
+on:
+  workflow_dispatch:
+    inputs:
+      removeDeployment:
+        type: boolean
+        description: 'Remove deployed module'
+        required: false
+        default: true
+      prerelease:
+        type: boolean
+        description: 'Publish prerelease module'
+        required: false
+        default: false
+  push:
```

Comment:
In general I'd recommend to recreate this and the other file based on any of the other pipelines in latest upstream CARML. We introduced a few new parameters, templates and other cool improvements :)